bower.json

```
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-narrative",
-  "version": "6.1.0",
+  "version": "6.1.1",
```

As this is adding a new feature...

```suggestion
  "version": "6.2.0",
```